.circleci/config.yml

```
@@ -337,6 +337,20 @@ jobs:
       - restore_node_modules
       - restore_built_packages
       - run: npm run test --silent -- --packages << parameters.package >> --browsers defaults
+      - run:
+          name: 'Check if tap test failed'
+          command: |
+            mkdir -p reports/junit/karma
+            FAILURES=$(find reports/junit/karma/ -iname '*.xml' -print0 | xargs -0 sed -n 's/.*failures=\"\([^\"]*\).*/\1/p' | xargs)
+
+            echo Failures $FAILURES
+
+            if [[ $FAILURES =~ ^([0 ])*$ ]]; then
+              echo 'Tap test succeeded'
```

Would be nice to have the result have the package it was testing echoed as well:

```suggestion
              echo 'Tap test for << parameters.package >> succeeded'
```

---

I added this to the commands being run, please let me know if these changes look okay.